.rust_alpha/rdsdata/src/bin/rdsdata-helloworld.rs

```diff
@@ -18,11 +18,11 @@ struct Opt {
     #[structopt(short, long)]
     query: String,
 
-    /// The ARN of your Aurora serverless DB cluster.
+    /// The Amazon Resource Name (ARN) of your Amazon Aurora serverless Amazon DynamoDB cluster.
```

Comment:
Aurora is not DynamoDB. The Aurora doc usage is:
Amazon Aurora DB cluster.

Reply:
Ah, gotcha.